app/views/manage_assessments/courses/show.html.erb

```diff
@@ -30,6 +29,21 @@
   </section>
 
   <section class="outcome-cards split-tab-panel" data-content="unmatched">
+    <div class="progress-outcomes container">
+      <span class="progress-outcomes-percent"
+            style="width: <%= progress_bar_percentage_width(@course.covered_outcomes_count,
+            @course.outcomes_count) %>">
+        <%= progress_bar_percentage_width(@course.covered_outcomes_count,
+            @course.outcomes_count) %>
+
```

Comment:
Gonna nitpick, but I feel like it's easier to read the block without this extra line